blobfile/_common.py

```diff
@@ -55,6 +56,9 @@
     "('Connection aborted.',",
 ]
 
+ACCESS_TOKEN_FILE = "~/.blobfile/azure_access_tokens.jsonl"

```

Comment:
Probably best to expanduser here so that it doesn't get accidentally forgotten elsewhere.

Reply:
Yea, that seems cleaner.  I'll do that